### ISSUE TITLE
3.104.1 - scope is not identical to ITI-8

### ISSUE THREAD

**Issue by lynnfel** [IHE/ITI.PIXm#80]
**Section Number** 3.104.1

**Issue** We say "The scope is identical to ITI TF-2: 3.8.1 ."   But, ITI-104 has Remove, and ITI-8 does not

**Proposed Change** 
If the scope is identical, then get rid of Remove in 104
If the scope is not identical, then fix this scope section.

**Priority**:
- Medium: Significant issue or clarification. Requires discussion, but should not lead to long debate.

**Comment by oliveregger** [comment]
agree, ITI-104 has Remove, and ITI-8 does not, therefore we need to fix the scope section.

**Comment by oliveregger** [comment]
ci pr http://build.fhir.org/ig/IHE/ITI.PIXm/branches/scope_differ_iti8/ITI-104.html

**Comment by lynnfel** [comment]
I think this Issue can be closed with no change to the scope statement in ITI-104

The Scope statement for ITI-104 differs from ITI-8 and ITI-44 because there is an additional sentence due to ITI-104 having a remove message.
